src/pages/søknad/steg/bo-og-opphold-i-norge/Bo-og-opphold-i-norge.tsx

```diff
@@ -330,6 +368,59 @@ const BoOgOppholdINorge = (props: { forrigeUrl: string; nesteUrl: string }) => {
                                 feil={formik.errors.ektefellePartnerSamboer}
                             />
                         )}
+
+                        <RadioGruppe
+                            legend={intl.formatMessage({ id: 'input.adresse.tittel' })}
+                            feil={formik.errors.borPåAdresse}
+                        >
+                            <p className={styles.undertittelForAdresse}>
+                                {intl.formatMessage({ id: 'input.adresse.undertittel' })}
```

Comment:
Kanskje det funker med
```suggestion
                            feil={formik.errors.borPåAdresse}
                            description={intl.formatMessage({ id: 'input.adresse.undertittel' })}
                        >
                                
```
?